### ISSUE TITLE
Screenreader cannot read TreeTable rows as expected

### ISSUE THREAD

**Issue by muhtomakin** [primefaces/primereact#6744]
### Describe the bug

With treetable component screenreader cannot read the focused row correctly. It reads like "text frame expanded 3 of 11 level 1", which is unexpected behavior.

### PrimeReact version

10.6.6

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

Firefox

### Expected behavior

With chrome screenreader works as expected. It reads what has got the row exactly.
